packages/web3-eth-contract/src/contract.ts

```diff
@@ -281,6 +351,7 @@ export class Contract<Abi extends ContractAbi>
 		abi: AbiFunctionFragment,
```

Comment:
`_contractMethodSend` should send `eth_sendTransaction` via functions in `eth` & `eth-tx` , for tracking could you create issue

Reply:
https://github.com/ChainSafe/web3.js/issues/4754